src/stirling/source_connectors/socket_tracer/socket_trace_connector.cc

```diff
@@ -95,8 +95,7 @@ DEFINE_int32(stirling_enable_nats_tracing, px::stirling::TraceMode::On,
              "If true, stirling will trace and process NATS messages.");
 DEFINE_int32(stirling_enable_kafka_tracing, px::stirling::TraceMode::On,
              "If true, stirling will trace and process Kafka messages.");
-DEFINE_int32(stirling_enable_mux_tracing,
-             gflags::Uint32FromEnv("PL_STIRLING_TRACER_ENABLE_MUX", px::stirling::TraceMode::Off),
```

Comment:
Removing this environment variable was needed in order to get the `Init()` function working in the `mux_trace_bpf_test.cc` and `thriftmux_openssl_trace_bpf_test.cc`. It also aligns the mux flag with how the others are managed.